src/app/static/package.json

```diff
@@ -56,5 +57,8 @@
     "webpack": "^4.46.0",
     "webpack-node-externals": "^1.7.2",
     "webpack-stream": "^5.2.1"
+  },
+  "dependencies": {
+    "vue-select": "^3.11.2"
```

Comment:
All our other deps are (for better or worse) in `devDependencies`, so let's be consistent